resources/virt.go

```diff
@@ -0,0 +1,521 @@
+package resources
+
+import (
```

Comment:
Imports should be split into three chunks (with a newline between each) first chunk normal imports, 2nd internal purpleidea/mgmt/ imports, third chunk external imports like the golang virt lib. Eg: https://github.com/purpleidea/mgmt/blob/master/resources/svc.go#L22